cpp/src/arrow/io/caching.h

```diff
@@ -41,7 +41,12 @@ struct ARROW_EXPORT CacheOptions {
   ///   combining two consecutive ranges would produce a range of a
   ///   size greater than this, they are not combined
   int64_t range_size_limit;
-  /// \brief A lazy cache does not perform any I/O until requested.
+  /// \brief A lazy cache does not perform any I/O until requested. 
+  ///   lazy = false: request all byte ranges when PreBuffer or WillNeed is called.
+  ///   lazy = True, prefetch_limit = 0: request merged byte ranges when the reader needs 
+  ///   only after the reader needs them. 
```

Comment:
The "when the reader needs only after the reader needs them" part doesn't sound as a fully correct sentence

Reply:
Oops

Reply:
Sorry about that. Its fixed now.